pkg/utils/printer/printer.go

```diff
@@ -99,6 +101,32 @@ func (p *JSONPrinter) Print(v interface{}) error {
 	return nil
 }
 
+func (p *JSONPrinter) Infof(format string, a ...interface{}) {
+	flag := viper.GetBool(config.ArgVerbose)
+	var toPrint = ToPrint{}
+	if flag {
+		str := fmt.Sprintf("[INFOF] "+format, a...)
+		toPrint.Message = str
+		err := WriteJSON(&toPrint, p.Stdout)
+		if err != nil {
+			return
+		}
+	}
```

Comment:
This code is duplicated in `Info()`. You can simply build your message here and then call `Info()`.

Reply:
removed

Reply:
done